### ISSUE TITLE
Define CTA for "how to apply"

### ISSUE THREAD

**Issue by diegocob** [GSA/px-bears-drupal#260]

**Comment by diegocob** [comment]
Creating this task to decide how we want to label the CTA's in the accordions. Need to make sure it's accessible. Putting "How to apply" and labeling "how to apply to [benefit_name]" is an option to keep them distinguishable for screen readers but visually appealing for folks that are sighted.  

![Image](https://github.com/GSA/px-bears-drupal/assets/125094772/c11f7bce-76db-422e-aaa0-08320cbe344a)



Ernie: 2023-07-31: Diego, did you have any thoughts about the CTA text in the benefit accordions? I'm not sure if the "Apply for [benefit title]" format we're currently using in the prototypes is final.

**Comment by amycole501** [comment]
Do we know if these CTAs are to be coded as buttons or links? The screen reader will announce whatever is inside the CTA if it's coded as a link or a button. 

If it's a link that opens in a new window, the screen reader will need to say "link opens in a new window" or "link will open the Social Security website" or something along those lines in the behind-the-scenes label.

From my perspective, if this is going to take someone somewhere offsite and not activate an application form right where they are OR start the application process immediately, I recommend the CTA to be a link and not a button. if we change the button look to a link it may reduce visitors' confusion that they are about to apply for the benefit right now.

https://medium.com/@h_locke/links-vs-buttons-e8b523660fb3

As for the wording, we want to keep it with an action word. So "Learn how to apply" makes more sense (even if it's a button or a link) to indicate they have more work to do offsite.

**Comment by diegocob** [comment]
Thanks for the comments, Amy. Is there a way to make it a link but have it
look like a button or any other design that makes it visually appealing?
This is the main button to take people to apply for the benefit and if we
keep it just like a link without any other distinction it may not be
convert as many user.

**Comment by amycole501** [comment]
I agree a link isn't as visually compelling. Here's some [information about this conundrum ](https://medium.com/simple-human/but-sometimes-links-look-like-buttons-and-buttons-look-like-links-9b371c57b3d2)  and a list of other sources here 
http://www.webaxe.org/proper-use-buttons-links/

Primarily we want to be clear they're leaving our site to start a journey elsewhere. I think that can be handled in a visually appealing and clearly worded way. My goal is to ensure we code it as a link (if it is indeed a link).

**Comment by diegocob** [comment]
Adding @fongcindy, @scottqueen-bixal, and @gchi25 to the thread on the "how to apply" CTAs in the results page. They can tell us how they are thinking of building it and how it can includ what you mention, Amy.

**Comment by amycole501** [comment]
We could create an image that looks like a button but is functionally a link. [W3C has some guidance here ](https://www.w3.org/WAI/tutorials/images/functional/#example-4-stand-alone-icon-image-that-has-a-function)for these types of "standalone links". Or an icon alongside a link to make the link stand out.

One more thing to consider:  Will that link take someone offsite and will that link open in a new window? If so, we need to ensure that we have a visible icon that indicates the visitor is about to open a new window. We could also handle it in the wording. "Apply on the social security website" 

[More examples here](https://helios.hashicorp.design/components/link/standalone)

![image](https://github.com/GSA/px-bears-drupal/assets/117778073/1177b4d7-d2b2-44d6-83e7-3a21f27ab570)

[Article](https://medium.com/simple-human/but-sometimes-links-look-like-buttons-and-buttons-look-like-links-9b371c57b3d2):
"Perhaps, if users knew they could open a call to action in a new window, they would? I certainly might. Maybe others wouldn’t. But does that matter? [Choice](https://inclusivedesignprinciples.org/#offer-choice) is a principle of inclusive design."
